samples/charts/financial-chart/annotations/Pages/FinancialChartAnnotations.razor

```diff
@@ -0,0 +1,182 @@
+                   checked=@CrosshairsVisible
+                   onchange=OnCrosshairsVisible /> Crosshair
+        </label>
+        <label className="igOptions-item">
+            <input type="checkbox"
+                   checked=@CalloutsVisible
+                   onchange=OnCalloutsVisible /> Callouts
+        </label>
+        <label className="igOptions-item">
+            <input type="checkbox"
+                   checked=@FinalValuesVisible
+                   onchange=OnFinalValuesVisible /> Final Values
+        </label>
+        <label className="igOptions-item">
+            <input type="checkbox"
+                   checked=@MarkersVisible
+                   onchange=OnMarkersVisible /> Markers
+        </label>
+    </div>
+
+    <div class="igComponent">
+
+        @* NOTE making sure data is loaded before we can render it *@
+        @if (DataSource != null)
+        {
+            <FinancialChart Width="100%"
+                            Height="100%"
+                            ChartType=FinancialChartType.Line
+                            Thickness=2
+                            ExcludedProperties="ExcludedProperties"
+                            XAxisMode="XAxisMode"
+                            YAxisMode="YAxisMode"
+                            DataSource="DataSource"
+                            
+                            CalloutsVisible="CalloutsVisible"
+                            CalloutsXMemberPath="Index"
+                            CalloutsYMemberPath="Value"
+                            CalloutsLabelMemberPath="Info"
+                            CalloutsContentMemberPath="Info"
+                            CrosshairsSnapToData=false
+                            CrosshairsDisplayMode="CrosshairsMode"
+                            CrosshairsAnnotationEnabled="CrosshairsVisible"
+                            FinalValueAnnotationsVisible="FinalValuesVisible" />
+        }
+
+    </div>
+</div>
+
+@* NOTE you can change this C# code to affect behaviour of this page: *@
+@code {
+
+    public string[] ExcludedProperties;
+    public FinancialChartXAxisMode XAxisMode;
+    public FinancialChartYAxisMode YAxisMode;
+    Samples.Services.Stock[] DataSource;
+    public Boolean CalloutsVisible;
+    public String Index;
+    public string Value;
+    public CrosshairsDisplayMode CrosshairsMode;
+    public Boolean CrosshairsVisible;
+    public Boolean FinalValuesVisible;
+    public Boolean MarkersVisible;
+    public MarkerType MarkerTypes;
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        // Console.WriteLine("Sample runtime: " + InfragisticsBlazor.JsRuntime);
+        Console.WriteLine("Sample OnInitialized...");
+        FinancialChartModule.Register(InfragisticsBlazor);
+        initData();
+    }
+
+    protected void OnCrosshairsVisible(CheckedChangedEventArgs e)
+    {
+        var isVisible = e.IsChecked;
+        CrosshairsVisible = isVisible;
```

Comment:
remove `isVisible` and use `CrosshairsVisible = e.IsChecked`
also use `CrosshairsVisible` in the IF statement